[2025-06-02T11:18:22.329+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: movie_recommendation_pipeline.cleanup_all manual__2025-06-02T11:17:55.612538+00:00 [queued]>
[2025-06-02T11:18:22.336+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: movie_recommendation_pipeline.cleanup_all manual__2025-06-02T11:17:55.612538+00:00 [queued]>
[2025-06-02T11:18:22.337+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2025-06-02T11:18:22.348+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): cleanup_all> on 2025-06-02 11:17:55.612538+00:00
[2025-06-02T11:18:22.355+0000] {standard_task_runner.py:60} INFO - Started process 2980 to run task
[2025-06-02T11:18:22.358+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'movie_recommendation_pipeline', 'cleanup_all', 'manual__2025-06-02T11:17:55.612538+00:00', '--job-id', '1690', '--raw', '--subdir', 'DAGS_FOLDER/mlops_movie_dag.py', '--cfg-path', '/tmp/tmpg5jc2pda']
[2025-06-02T11:18:22.359+0000] {standard_task_runner.py:88} INFO - Job 1690: Subtask cleanup_all
[2025-06-02T11:18:22.393+0000] {task_command.py:423} INFO - Running <TaskInstance: movie_recommendation_pipeline.cleanup_all manual__2025-06-02T11:17:55.612538+00:00 [running]> on host 6520fa36f146
[2025-06-02T11:18:22.452+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='movie_recommendation_pipeline' AIRFLOW_CTX_TASK_ID='cleanup_all' AIRFLOW_CTX_EXECUTION_DATE='2025-06-02T11:17:55.612538+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-02T11:17:55.612538+00:00'
[2025-06-02T11:18:22.509+0000] {mlops_movie_dag.py:12} INFO - üü¶ Running command: dvc remove /opt/***/models/hybrid_model.pkl.dvc
[2025-06-02T11:18:23.184+0000] {mlops_movie_dag.py:19} INFO - üü© [stdout]:

[2025-06-02T11:18:23.185+0000] {mlops_movie_dag.py:21} WARNING - üü® [stderr]:
ERROR: '/opt/***/models/hybrid_model.pkl.dvc' does not exist

[2025-06-02T11:18:23.185+0000] {mlops_movie_dag.py:28} ERROR - ‚ùå Subprozess-Fehler (exit code 1): dvc remove /opt/***/models/hybrid_model.pkl.dvc
[2025-06-02T11:18:23.186+0000] {mlops_movie_dag.py:31} ERROR - ‚ùå Subprozess-Ausnahme: Command '['dvc', 'remove', '/opt/***/models/hybrid_model.pkl.dvc']' returned non-zero exit status 1.
[2025-06-02T11:18:23.186+0000] {mlops_movie_dag.py:90} WARNING - Ignoriere Fehler bei dvc remove f√ºr /opt/***/models/hybrid_model.pkl.dvc: Command '['dvc', 'remove', '/opt/***/models/hybrid_model.pkl.dvc']' returned non-zero exit status 1.
[2025-06-02T11:18:23.249+0000] {mlops_movie_dag.py:12} INFO - üü¶ Running command: dvc remove /opt/***/data/processed/hybrid_matrix.csv.dvc
[2025-06-02T11:18:23.662+0000] {mlops_movie_dag.py:19} INFO - üü© [stdout]:

[2025-06-02T11:18:23.663+0000] {mlops_movie_dag.py:21} WARNING - üü® [stderr]:
ERROR: '/opt/***/data/processed/hybrid_matrix.csv.dvc' does not exist

[2025-06-02T11:18:23.663+0000] {mlops_movie_dag.py:28} ERROR - ‚ùå Subprozess-Fehler (exit code 1): dvc remove /opt/***/data/processed/hybrid_matrix.csv.dvc
[2025-06-02T11:18:23.664+0000] {mlops_movie_dag.py:31} ERROR - ‚ùå Subprozess-Ausnahme: Command '['dvc', 'remove', '/opt/***/data/processed/hybrid_matrix.csv.dvc']' returned non-zero exit status 1.
[2025-06-02T11:18:23.664+0000] {mlops_movie_dag.py:90} WARNING - Ignoriere Fehler bei dvc remove f√ºr /opt/***/data/processed/hybrid_matrix.csv.dvc: Command '['dvc', 'remove', '/opt/***/data/processed/hybrid_matrix.csv.dvc']' returned non-zero exit status 1.
[2025-06-02T11:18:23.671+0000] {mlops_movie_dag.py:12} INFO - üü¶ Running command: dvc remove /opt/***/data/processed/hybrid_matrix_features.txt.dvc
[2025-06-02T11:18:24.065+0000] {mlops_movie_dag.py:19} INFO - üü© [stdout]:

[2025-06-02T11:18:24.066+0000] {mlops_movie_dag.py:21} WARNING - üü® [stderr]:
ERROR: '/opt/***/data/processed/hybrid_matrix_features.txt.dvc' does not exist

[2025-06-02T11:18:24.067+0000] {mlops_movie_dag.py:28} ERROR - ‚ùå Subprozess-Fehler (exit code 1): dvc remove /opt/***/data/processed/hybrid_matrix_features.txt.dvc
[2025-06-02T11:18:24.067+0000] {mlops_movie_dag.py:31} ERROR - ‚ùå Subprozess-Ausnahme: Command '['dvc', 'remove', '/opt/***/data/processed/hybrid_matrix_features.txt.dvc']' returned non-zero exit status 1.
[2025-06-02T11:18:24.067+0000] {mlops_movie_dag.py:90} WARNING - Ignoriere Fehler bei dvc remove f√ºr /opt/***/data/processed/hybrid_matrix_features.txt.dvc: Command '['dvc', 'remove', '/opt/***/data/processed/hybrid_matrix_features.txt.dvc']' returned non-zero exit status 1.
[2025-06-02T11:18:24.075+0000] {mlops_movie_dag.py:12} INFO - üü¶ Running command: dvc remove /opt/***/models/user_model.pkl.dvc
[2025-06-02T11:18:24.459+0000] {mlops_movie_dag.py:19} INFO - üü© [stdout]:

[2025-06-02T11:18:24.460+0000] {mlops_movie_dag.py:21} WARNING - üü® [stderr]:
ERROR: '/opt/***/models/user_model.pkl.dvc' does not exist

[2025-06-02T11:18:24.461+0000] {mlops_movie_dag.py:28} ERROR - ‚ùå Subprozess-Fehler (exit code 1): dvc remove /opt/***/models/user_model.pkl.dvc
[2025-06-02T11:18:24.461+0000] {mlops_movie_dag.py:31} ERROR - ‚ùå Subprozess-Ausnahme: Command '['dvc', 'remove', '/opt/***/models/user_model.pkl.dvc']' returned non-zero exit status 1.
[2025-06-02T11:18:24.462+0000] {mlops_movie_dag.py:90} WARNING - Ignoriere Fehler bei dvc remove f√ºr /opt/***/models/user_model.pkl.dvc: Command '['dvc', 'remove', '/opt/***/models/user_model.pkl.dvc']' returned non-zero exit status 1.
[2025-06-02T11:18:24.475+0000] {mlops_movie_dag.py:12} INFO - üü¶ Running command: dvc remove /opt/***/data/processed/user_matrix.csv.dvc
[2025-06-02T11:18:24.871+0000] {mlops_movie_dag.py:19} INFO - üü© [stdout]:

[2025-06-02T11:18:24.871+0000] {mlops_movie_dag.py:21} WARNING - üü® [stderr]:
ERROR: '/opt/***/data/processed/user_matrix.csv.dvc' does not exist

[2025-06-02T11:18:24.872+0000] {mlops_movie_dag.py:28} ERROR - ‚ùå Subprozess-Fehler (exit code 1): dvc remove /opt/***/data/processed/user_matrix.csv.dvc
[2025-06-02T11:18:24.872+0000] {mlops_movie_dag.py:31} ERROR - ‚ùå Subprozess-Ausnahme: Command '['dvc', 'remove', '/opt/***/data/processed/user_matrix.csv.dvc']' returned non-zero exit status 1.
[2025-06-02T11:18:24.873+0000] {mlops_movie_dag.py:90} WARNING - Ignoriere Fehler bei dvc remove f√ºr /opt/***/data/processed/user_matrix.csv.dvc: Command '['dvc', 'remove', '/opt/***/data/processed/user_matrix.csv.dvc']' returned non-zero exit status 1.
[2025-06-02T11:18:24.879+0000] {mlops_movie_dag.py:12} INFO - üü¶ Running command: dvc remove /opt/***/data/processed/user_matrix_features.txt.dvc
[2025-06-02T11:18:25.274+0000] {mlops_movie_dag.py:19} INFO - üü© [stdout]:

[2025-06-02T11:18:25.274+0000] {mlops_movie_dag.py:21} WARNING - üü® [stderr]:
ERROR: '/opt/***/data/processed/user_matrix_features.txt.dvc' does not exist

[2025-06-02T11:18:25.275+0000] {mlops_movie_dag.py:28} ERROR - ‚ùå Subprozess-Fehler (exit code 1): dvc remove /opt/***/data/processed/user_matrix_features.txt.dvc
[2025-06-02T11:18:25.275+0000] {mlops_movie_dag.py:31} ERROR - ‚ùå Subprozess-Ausnahme: Command '['dvc', 'remove', '/opt/***/data/processed/user_matrix_features.txt.dvc']' returned non-zero exit status 1.
[2025-06-02T11:18:25.276+0000] {mlops_movie_dag.py:90} WARNING - Ignoriere Fehler bei dvc remove f√ºr /opt/***/data/processed/user_matrix_features.txt.dvc: Command '['dvc', 'remove', '/opt/***/data/processed/user_matrix_features.txt.dvc']' returned non-zero exit status 1.
[2025-06-02T11:18:25.276+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-06-02T11:18:25.285+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=movie_recommendation_pipeline, task_id=cleanup_all, execution_date=20250602T111755, start_date=20250602T111822, end_date=20250602T111825
[2025-06-02T11:18:25.300+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-02T11:18:25.319+0000] {taskinstance.py:3280} INFO - 3 downstream tasks scheduled from follow-on schedule check
